pommerman/graphics.py

```diff
@@ -0,0 +1,327 @@
+from gym.envs.classic_control import rendering
+from gym.utils import reraise
+from random import randint
+import numpy as np
+from time import strftime
+import math
+import os
+
+try:
+    import pyglet
+except ImportError as e:
+    reraise(suffix="HINT: you can install pyglet directly via 'pip install pyglet'. But if you really just want to install all Gym dependencies and not have to think about it, 'pip install -e .[all]' or 'pip install gym[all]' will do it.")
+
+try:
+    from pyglet.gl import *
+except ImportError as e:
+    reraise(prefix="Error occured while running `from pyglet.gl import *`",suffix="HINT: make sure you have OpenGL install. On Ubuntu, you can run 'apt-get install python-opengl'. If you're running on a server, you may need a virtual frame buffer; something like this should work: 'xvfb-run -s \"-screen 0 1400x900x24\" python <your_script.py>'")
+
+
+IMAGES_DICT = {
```

Comment:
file_names = ['Passage', 'Rigid', ..., 'X-No-Background']
IMAGES_DICT = {num: {'id': num, 'file_name': '%s.png' % file_name, 'name': file_name, 'image': None} for num, file_name in enumerate(file_names)}

Reply:
This type of stuff in python still gets me. ha. It is really neat to see.